elements/rh-announcement/rh-announcement.ts

```diff
@@ -0,0 +1,123 @@
+import { SlotController } from '@patternfly/pfe-core/controllers/slot-controller.js';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { LitElement, html, svg, type PropertyValues } from 'lit';
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
+import { colorContextProvider } from '../../lib/context/color/provider.js';
+
+import styles from './rh-announcement.css';
+/**
+ * Announcements are flexible surfaces used to group information in a full width banner layout, traditionally across the top of a page. 
+ * They are used to announce new features, promos, or news. 
+ * @summary     Arranges content and interactive elements in a layout
+ * @slot        image
+ *              If this slot is used, we expect an image tag with a width and height set.
+ *              An icon, svg, or use of the icon component are also valid in this region.
+ * @slot        Any content that is not designated for the header or footer slot, will go to this slot.
+ * @slot        cta
+ *              If this slot is used, we expect a rh-cta component.
```

Comment:
If you view the docs for rh-announcement and click on Code, scroll down to the table, no docs are showing up in the "Slots" section of the table. Maybe some formatting tweaking is required here?